src/screens/Departures/state/quay-state.ts

```diff
@@ -355,7 +362,7 @@ async function fetchEstimatedCalls(
       id: quay.id,
       startTime: queryInput.startTime,
       numberOfDepartures: queryInput.numberOfDepartures,
-      timeRange: timeRange,
+      timeRange: queryInput.timeRange,
```

Comment:
Remove the existing `timeRange` variable?

Reply:
I dint know that you had commented,  I have force pushed this fix to previous commit itself 😅